config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -174,3 +174,198 @@ periodics:
       # fuse needs privileged mode
       securityContext:
         privileged: true
+- name: ci-etcd-performance-ratio-1-128-amd64
+  interval: 24h
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: main
+  annotations:
+    testgrid-dashboards: sig-etcd-periodics
+    testgris-tab-name: ci-etcd-performance-amd64
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240310-bac0a3c105-master
+      command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        apt update && apt-get --yes install cmake libfuse3-dev libfuse3-3 fuse3
+        sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf
+        make install-lazyfs
+        set -euo pipefail
+        make build tools
+        RATIO_LIST=1/128 ./tools/rw-heatmaps/rw-benchmark.sh
+        cp result-*.csv "${ARTIFACTS}/result-$(git rev-parse HEAD).csv"
+      resources:
+        requests:
+          cpu: "7"
+          memory: "4Gi"
+        limits:
+          cpu: "7"
+          memory: "4Gi"
+      # fuse needs privileged mode
+      securityContext:
+        privileged: true
```

Comment:
As above - Can drop privileged as that is only required for lazyfs fuse for robustness tests.

Reply:
Done